System.IO.Abstractions.TestingHelpers.Tests/MockFileStreamTests.cs

```diff
@@ -74,5 +74,20 @@ public void MockFileStream_Constructor_ReadTypeNotWritable()
             Assert.IsFalse(stream.CanWrite);
             Assert.Throws<NotSupportedException>(() => stream.WriteByte(1));
         }
+
+        [Test]
+        public void MockFileStream_Close_MultipleCallsDontThrow()
+        {
+            var fileSystem = new MockFileSystem();
+            var path = XFS.Path("C:\\test");
+            fileSystem.AddFile(path, new MockFileData("Bla"));
+            var stream = fileSystem.File.OpenRead(path);
+
+            // Act
+            stream.Close();
+
+            // Assert
+            Assert.DoesNotThrow(() => stream.Close());
```

Comment:
Is this the same behavior that a real `FileStream` shows?

Reply:
Yes. These would pass:

``` CS

        [Test]
        public void DefaultFileStream_Close_MultipleCallsDontThrow()
        {
            var tempPath = Path.GetTempPath();
            var fileStream = File.Create(Path.Combine(tempPath, "so_i_can_see_it_close"));

            // Act
            fileStream.Close();

            // Assert
            Assert.DoesNotThrow(() => fileStream.Close());
        }

        [Test]
        public void DefaultFileStream_Dispose_MultipleCallsDontThrow()
        {
            var tempPath = Path.GetTempPath();
            var fileStream = File.Create(Path.Combine(tempPath, "so_i_can_see_it_dispose"));

            // Act
            fileStream.Dispose();

            // Assert
            Assert.DoesNotThrow(() => fileStream.Dispose());
        }
```